assets/app/styles/_component-animations.less

```diff
@@ -20,3 +20,130 @@
   overflow: hidden;
   .transition(height .10s ease);
 }
+
+// Ellipsis Loader
```

Comment:
After removing the prefixed styles, I'm not seeing the generated prefixed rules.